core/src/lib.rs

```diff
@@ -203,6 +203,14 @@ mod tests {
 
     fn test_proof_params() -> HashMap<String, Value> {
         let mut prover_args = HashMap::new();
+        prover_args.insert(
+            "native".to_string(),
+            json! {
+                {
+                    "write_guest_input_path": "./provers/sp1/contracts/src/fixtures/input.json"
+                }
+            },
```

Comment:
```suggestion
```
Should we serialize to this path here? If not would not serialize it. Or if we do (for testing purposes so this code path doesn't break) we could serialize to some general test dir that is git ignored.

Reply:
changed to null